### PR TITLE
Optimize Printing Experience

### LINKED ISSUE #1502: Add print style support
Recently I would like to export *.PDF from my nextra site, however, a problem appears that it's still using the web style. 
Is there any way to add a print style change so that the PDF will be better for reading?
e.g.: remove the nav bar, change the footer position, etc.
Many thanks.

### PR DESCRIPTION
**Pull request by MrWillCom**
Hide some unnecessary parts of the page, like navbar, sidebar, when being printed. fixes #1502

### CODE REVIEW

File: packages/nextra-theme-docs/src/components/footer.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { renderComponent } from '../utils'
 export function Footer({ menu }: { menu?: boolean }): ReactElement {
   const config = useConfig()
   return (
-    <footer className="nx-bg-gray-100 nx-pb-[env(safe-area-inset-bottom)] dark:nx-bg-neutral-900">
+    <footer className="nx-bg-gray-100 nx-pb-[env(safe-area-inset-bottom)] dark:nx-bg-neutral-900 print:nx-bg-transparent">
```
Comment (B2o5T) [inline packages/nextra-theme-docs/src/components/footer.tsx:11]: I guess we can hide the footer while printing, also maybe we should hide the breadcrumb also?

And I like the idea of using `print:*` classes!

Comment (MrWillCom) [reply packages/nextra-theme-docs/src/components/footer.tsx:11]: In my opinion, I prefer save the footer and the breadcrumb. The footer tells the author and license information of the page. And the breadcrumb can be seen as the page header showing the current chapter. Plus, they are useful when the page is exported to PDF. Maybe we can simply make the footer take less area when being printed?

**Review by B2o5T — APPROVED** [review]
Could you add a changeset, so users will know about this improvement?

**Comment by MrWillCom** [comment]
👌 Sure! Added in 8d5b57d.

File: packages/nextra-theme-docs/src/components/footer.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export function Footer({ menu }: { menu?: boolean }): ReactElement {
       <div
         className={cn(
           'nx-mx-auto nx-flex nx-max-w-[90rem] nx-justify-center nx-py-12 nx-text-gray-600 dark:nx-text-gray-400 md:nx-justify-start',
-          'nx-pl-[max(env(safe-area-inset-left),1.5rem)] nx-pr-[max(env(safe-area-inset-right),1.5rem)]'
+          'nx-pl-[max(env(safe-area-inset-left),1.5rem)] nx-pr-[max(env(safe-area-inset-right),1.5rem)] print:nx-py-2'
```
Comment (B2o5T) [inline packages/nextra-theme-docs/src/components/footer.tsx:27]:
```suggestion
          'nx-pl-[max(env(safe-area-inset-left),1.5rem)] nx-pr-[max(env(safe-area-inset-right),1.5rem)]'
```

**Review by B2o5T — APPROVED** [review]

**Pull request merged**